src/main/java/wolox/training/security/SecurityConfig.java

```diff
@@ -0,0 +1,35 @@
+package wolox.training.security;
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.

Reply:
Please review again, this branch wasn't update with main.